### PR TITLE
Cleanup DB warnings for nullable reference types

### PR DESCRIPTION
**Pull request by sslaws**

### CODE REVIEW

**Comment by healthgatewaybc** [comment]
Patient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=patientservice_pr_287)

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_287)

**Comment by healthgatewaybc** [comment]
Immunization Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=immunization_pr_287)

File: Apps/Database/src/Delegates/DBDrugLookupDelegate.cs [hunk 1]
```diff
@@ -78,7 +78,7 @@ public List<PharmaCareDrug> GetPharmaCareDrugsByDIN(List<string> drugIdentifiers
         /// <inheritdoc/>
         public Dictionary<string, string> GetDrugsBrandNameByDIN(List<string> drugIdentifiers)
         {
-            Contract.Requires(drugIdentifiers != null);
+            // Contract.Requires(drugIdentifiers != null);
```
Comment (TiagoGraf89) [inline Apps/Database/src/Delegates/DBDrugLookupDelegate.cs]: Nuke this

Comment (FuriousLlama) [reply Apps/Database/src/Delegates/DBDrugLookupDelegate.cs]: +1

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_287)

**Comment by healthgatewaybc** [comment]
Immunization Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=immunization_pr_287)

**Pull request merged**